packages/atomic/src/components/atomic-result-list/atomic-result-list.tsx

```diff
@@ -113,7 +113,7 @@ export class AtomicResultList implements InitializableComponent {
   private get results() {
     return this.resultListState.results.map((result) => (
       <atomic-result
-        key={`${result.raw.permanentid}${this.resultListState.searchUid}`}
```

Comment:
Should we deprecate the `searchUid` property on the result list state? IIRC, we exposed the property to address this problem, but we are now no longer using it?